apps/namadillo/src/utils/dates.ts

```diff
@@ -0,0 +1,54 @@
+import * as fns from "date-fns";
+import { DateTime } from "luxon";
+
+export const epochToString = (epoch: bigint): string =>
+  `Epoch ${epoch.toString()}`;
+
+export const secondsToDateTime = (seconds: bigint): DateTime =>
+  DateTime.fromSeconds(Number(seconds));
+
+export const secondsToTimeString = (seconds: bigint): string =>
+  secondsToDateTime(seconds).toLocaleString(DateTime.TIME_SIMPLE);
+
+export const secondsToDateString = (seconds: bigint): string =>
+  secondsToDateTime(seconds).toLocaleString(DateTime.DATE_MED);
+
+export const secondsToDateTimeString = (seconds: bigint): string =>
+  `${secondsToDateString(seconds)}, ${secondsToTimeString(seconds)}`;
+
+export const secondsToFullDateTimeString = (seconds: bigint): string =>
+  secondsToDateTime(seconds).toLocaleString(DateTime.DATETIME_FULL);
+
+export const secondsToTimeRemainingString = (
+  startTimeInSeconds: bigint,
+  endTimeInSeconds: bigint
+): string | undefined => {
+  // TODO is this useful?
+  return;
```

Comment:
I would say this isn't useful, maybe we should delete any references to this function - it's been immediately returning for many months now :D

Reply:
Yeah, I was thinking the same!

But I explored the feature and Figma and realized this is related to the remaining time for the ongoing proposal. So it was an one line change and I did it already  😁 

![Screenshot 2024-12-20 at 12 09 12](https://github.com/user-attachments/assets/4314fb31-da31-4b7a-8d41-205245da7409)